client/src/utils/sortHelper.js

```diff
@@ -0,0 +1,23 @@
+import moment from 'moment';
+
+function filterDatesHelper(a, b) {
+    const momentA = moment(a.release_date, 'YYYY-MM-DD');
+    const momentB = moment(b.release_date, 'YYYY-MM-DD');
+
+    return momentB - momentA;
+}
+
+const sortHelper = (movies, sortField) => (
+    [...movies].sort((a, b) => {
```

Comment:
I'd recommend to use `reselect` lib for your data slices. For now your approach recalculates movies array on each action dispatch, which can cause performance issues in a long term

Reply:
Selector for sorting movies has been added